authorization/client.go

```diff
@@ -92,7 +92,7 @@ func insertMetadata(ctx context.Context, mds ...metadata.MD) context.Context {
 }
 
 func verifyTokenFormat(bearerToken string) error {
-	_, err := jwt.ParseString(bearerToken, jwt.WithValidate(true), jwt.WithAcceptableSkew(time.Second))
+	_, err := jwt.ParseString(bearerToken, jwt.WithVerify(false), jwt.WithAcceptableSkew(time.Second))
```

Comment:
By default Validate and Verification is enabled in V2. As this is here just to validate, we need to disable verification. Otherwise private key has to be provided